neptune/new/handler.py

```diff
@@ -38,32 +40,49 @@
 from neptune.new.types.value_copy import ValueCopy
 
 if TYPE_CHECKING:
-    from neptune.new.run import Run
+    from neptune.new.attribute_container import AttributeContainer
+
+
+def check_protected_paths(fun):
+    @wraps(fun)
+    def inner_fun(self: "Handler", *args, **kwargs):
+        # pylint: disable=protected-access
+        for path in self.PROTECTED_PATHS:
+            if self._path == path:
```

Comment:
`it self._path in self.PROTECTED_PATHS:`?

Reply:
Shame on me...